library/pages/CustomizationPage/index.js

```diff
@@ -0,0 +1,210 @@
+                        label="DEFAULT THEME"
+                        className="react-rainbow-customization_top-container-tab"
+                    />
+                </Tabset>
+                <section className="react-rainbow-customization_container">
+                    <RenderIf isTrue={activeTabName === 'theming'}>
+                        <div>
+                            <h2 className="react-rainbow-customization_section-heading-2">
+                                Customize React Rainbow with your own palette. You can change the
+                                main colors on your palette brand, success, warning, error and more…
+                            </h2>
+                            <h2 className="react-rainbow-customization_section-heading-2">
+                                Overview
+                            </h2>
+                            <h3 className="react-rainbow-customization_section-heading">
+                                The theme specifies the color of the components, darkness of the
+                                surfaces, level of shadow, appropriate opacity of texts elements,
+                                etc. Themes let you apply a consistent tone to your app. It allows
+                                you to customize all design aspects of your project in order to meet
+                                the specific needs of your business or brand. To promote greater
+                                consistency between apps, light and dark theme types are available
+                                to choose from. By default, components use the light theme type.
+                            </h3>
+                            <h2 className="react-rainbow-customization_section-heading-2">
+                                Application
+                            </h2>
+                            <h3 className="react-rainbow-customization_section-heading">
+                                If you wish to customize the theme, you need to use the{' '}
+                                {'<Application />'} component in order to inject a theme into your
+                                application. However, this is optional; React Rainbow components
+                                come with a default theme. Application relies on the context feature
+                                of React to pass the theme down to the components, so you need to
+                                make sure that Application is a parent of the components you are
+                                trying to customize. You can learn more about this in the API
+                                section
+                            </h3>
+                            <br />
+
+                            <Playground
+                                code={examples[1].content}
+                                evalInContext={examples[1].evalInContext}
+                                key={`${codeRevision}/1`}
```

Comment:
I see codeRevision is 0 the you are passing as key 0/1 which is 0 and below where Playground is used again you are passing ```key={`${codeRevision}/3`}``` which is 0 again and then a warning saying:
```Each child in a list should have a unique "key" prop.```
is showed in browser console.